waspc/cli/src/Wasp/Cli/Command/Db/Start.hs

```diff
@@ -0,0 +1,129 @@
+module Wasp.Cli.Command.Db.Start
```

Comment:
The command is now in the `start` scope, shouldn't the module reflect that:

`Wasp.Cli.Command.Start.Db`

Reply:
I thought it is ok to leave it in Db.Start because I felt it logically belong there -> it deals with db, and it starts it. And I saw `wasp start db` as a UX/DX thing.
But I moved it now, I don't think it makes much difference at the end, and it is a bit confusing that all other commands reflect in their positioning how they are called from CLI but this one.